src/Microsoft.OData.Core/UriParser/SemanticAst/ODataPathExtensions.cs

```diff
@@ -177,51 +177,72 @@ public static string ToContextUrlPathString(this ODataPath path)
             StringBuilder pathString = new StringBuilder();
             PathSegmentToContextUrlPathTranslator pathTranslator = PathSegmentToContextUrlPathTranslator.DefaultInstance;
             ODataPathSegment priorSegment = null;
+            bool findOperationWithoutPath = false;

```

Comment:
>findOperationWithoutPath [](start = 17, length = 24)

maybe call "foundOperationWithoutPath" or "includesOperationWithoutPath" (to describe the scenario), or "writeContextAsType" (to describe what the code should do).

Reply:
Resolved.